application/messages/claim.go

```diff
@@ -83,9 +83,20 @@ func ClaimPreapprovedQueueMessage(tx *pop.Connection, claim models.Claim) {
 	claimItem.LoadItem(tx, false)
 
 	data["estimate"] = "$-"
-	data["deductible"] = domain.PercentString(claim.GetDeductibleRate(tx))
 	data["repairThreshold"] = domain.Env.RepairThresholdString
 
+	deductibleRate := domain.PercentString(claim.GetDeductibleRate(tx))
+	claim.ClaimItems[0].LoadItem(tx, false)
+	claim.ClaimItems[0].Item.LoadCategory(tx, false)
+	if minimumDeductible := claim.ClaimItems[0].Item.Category.MinimumDeductible; minimumDeductible > 0 {
+		data["deductible"] = fmt.Sprintf(
+			"%s (subject to %s minimum)",
+			deductibleRate,
+			api.Currency(minimumDeductible))
+	} else {
+		data["deductible"] = deductibleRate
+	}
```

Comment:
Alternatively, without `else` (could also keep `claimItem.Item.Category.MinimumDeductible` instead of a temporary variable too):

```suggestion
	data["deductible"] = domain.PercentString(claim.GetDeductibleRate(tx))
	claimItem.LoadCategory(tx, false)
	if minimumDeductible := claimItem.Item.Category.MinimumDeductible; minimumDeductible > 0 {
		data["deductible"] += fmt.Sprintf(" (subject to %s minimum)", api.Currency(minimumDeductible))
	}
```

Either way, I'd replace `claim.ClaimItems[0]` with `claimItem` and remove the load item as that is already done above.

Reply:
Put suggestion as new comment